src/Sylius/Bundle/MailerBundle/Sender/Adapter/TwigSwiftMailerAdapter.php

```diff
@@ -59,19 +55,18 @@ public function __construct(
      */
     public function send(EmailInterface $email, array $recipients, array $data = array())
     {
-        $data = $this->twig->mergeGlobals($data);
-
         if (null !== $email->getTemplate()) {
+            $data = $this->twig->mergeGlobals($data);
+
             $template = $this->twig->loadTemplate($email->getTemplate());
 
             $subject = $template->renderBlock('subject', $data);
```

Comment:
Yes but you replace replace deprecated features, it seems like the same ... not ?